### ISSUE TITLE
skipRemoteConfig Necessary?

### ISSUE THREAD

**Issue by rdegges** [stormpath/stormpath-node-config#45]
https://github.com/stormpath/stormpath-node-config/blob/c11f1f59d88e3e64f6abe4fa829e64fdc31ab6ad/lib/strategy/EnrichClientFromRemoteConfigStrategy.js#L81-L83

It looks like this was just added for testing convenience. Are we using this anywhere else? I'm working on the python version of this, and can't really seem to find a use for it.

Thinking about dropping support for it, but didn't want to break anything =)

**Comment by robertjd** [comment]
I'm pretty sure that we removed anything that depended on it, that was our intention.  I'd need to dig in to be certain, though.

**Comment by typerandom** [comment]
Yeah, this was really just a convenience thing for testing. It should probably be refactored :)
